crates/astria-sequencer/src/main.rs

```diff
@@ -24,6 +24,17 @@ async fn main() -> ExitCode {
         config = serde_json::to_string(&config).expect("serializing to a string cannot fail"),
         "initializing sequencer"
     );
+
+    #[cfg(feature = "mint")]
+    tokio::spawn(async {
+        let duration = std::time::Duration::from_secs(5);
+        loop {
+            eprintln!("MINT FEATURE IS ENABLED!");
```

Comment:
Please return `warn!` :-D

I wanted logging to stderr in case tracing is disabled, but also `warn!` if tracing is enabled but we end up using a a tracing subscriber that sends directly to e.g. opentelemetry.

Reply:
updated!